src/main/java/bio/terra/service/resourcemanagement/google/GoogleResourceManagerService.java

```diff
@@ -0,0 +1,208 @@
+package bio.terra.service.resourcemanagement.google;
+
+import bio.terra.common.AclUtils;
+import bio.terra.service.resourcemanagement.exception.GoogleResourceException;
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.googleapis.json.GoogleJsonResponseException;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.cloudresourcemanager.CloudResourceManager;
+import com.google.api.services.cloudresourcemanager.model.Binding;
+import com.google.api.services.cloudresourcemanager.model.GetIamPolicyRequest;
+import com.google.api.services.cloudresourcemanager.model.Policy;
+import com.google.api.services.cloudresourcemanager.model.Project;
+import com.google.api.services.cloudresourcemanager.model.SetIamPolicyRequest;
+import com.google.common.annotations.VisibleForTesting;
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.collections4.ListUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Component;
+
+@Component
+public class GoogleResourceManagerService {
```

Comment:
Generally speaking, I find it's a bit of a weird naming scheme.  I like the fact that today we have a project service.  Google resource is verrrrry general (why wouldn't buckets fall under that).  I'm not really sure what the point of moving it out of projectservice is

Reply:
Needed to be moved out to avoid a circular dependency between `BufferService` and `GoogleProjectService`

Reply:
Is it worth adding that info as a comment?